src/Sarif.Viewer.VisualStudio/ResultTextMarker.cs

```diff
@@ -106,7 +106,8 @@ public Region GetSourceLocation()
         {
             Region sourceLocation = new Region()
             {
-                Offset = m_region.Offset,
+                ByteOffset = m_region.ByteOffset,
+                CharOffset = m_region.CharOffset,
```

Comment:
I doubt you need this line. Before the change, this method was able to do everything it needed to do with `Offset` (which meant an offset measured in bytes). So it should still be able to do everything it needs to do with `ByteOffset`, which means the same thing that `Offset` used to mean.

Reply:
Was wavering on this. Fixed.